modules/check_in/app/controllers/check_in/v2/patient_check_ins_controller.rb

```diff
@@ -7,7 +7,8 @@ class PatientCheckInsController < CheckIn::ApplicationController
       after_action :after_logger, only: %i[show create]
 
       def show
-        check_in_session = CheckIn::V2::Session.build(data: { uuid: params[:id] }, jwt: low_auth_token)
+        check_in_session = CheckIn::V2::Session.build(data: { uuid: params[:id], handoff: handoff? },
```

Comment:
Pls add request specs for the changes in `patient_check_ins_controller`

Reply:
I feel that adding a parameter doesn't really need a separate test. We have a separate ticket to add request specs when we add more logic to the underlying service.